src/_gettsim/config.py

```diff
@@ -67,13 +67,36 @@ def set_array_backend(backend: str):
 
 SUPPORTED_GROUPINGS = {
     "hh": {
-        "name": "household",
-        "description": "all individuals living in the same household.",
+        "name": "Haushalt",
+        "description": "The relevant unit for Wohngeld. Encompasses more people than"
+        " the Bedarfsgemeinschaft (e.g., possibly more than 2 generations). Relevant"
+        " unit for Wohngeld. `vg` derives from Verantwortungs- und"
+        " Einstehensgemeinschaft, though this might be a misnomer.",
+        "is_endogenous": False,
```

Comment:
```suggestion
        "potentially_endogenous": False,
```

We can always override the own calculation by passing columns, so `is_` is a bit misleading.

Put differently, I will never be able to derive these columns from other variables.